js/binance.js

```diff
@@ -4147,21 +4147,27 @@ module.exports = class binance extends Exchange {
             'asset': currency['id'],
             'amount': this.currencyToPrecision (code, amount),
         };
-        request['type'] = this.safeString (params, 'type');
```

Comment:
We cannot simply drop this line, cause `if (request['type'] === undefined) {` will throw a KeyError exception in Python after transpilation: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#sanitizing-input-with-safe-methods

Reply:
uh oh, i've realized that atm.